go/common/rpc/converters.go

```diff
@@ -69,7 +69,7 @@ func ToRollupHeaderMsg(header *common.Header) *generated.HeaderMsg {
 	var headerMsg generated.HeaderMsg
 	withdrawalMsgs := make([]*generated.WithdrawalMsg, 0)
 	for _, withdrawal := range header.Withdrawals {
-		withdrawalMsg := generated.WithdrawalMsg{Amount: withdrawal.Amount, Recipient: withdrawal.Recipient.Bytes(), Contract: withdrawal.Contract.Bytes()}
+		withdrawalMsg := generated.WithdrawalMsg{Amount: withdrawal.Amount.Bytes(), Recipient: withdrawal.Recipient.Bytes(), Contract: withdrawal.Contract.Bytes()}
```

Comment:
protobuf doesn't support big.Int directly but we can easily read/write big.Ints from a protobuf `bytes` field.